README.md

```diff
@@ -73,15 +75,41 @@ Configuration lives in `~/.nixpkgs/darwin-configuration.nix`. Check out
 There is also preliminary support for building your configuration using a [flake](https://nixos.wiki/wiki/Flakes).  This
 is mostly based on the flake support that was added to NixOS.
 
-A minimal example of using an existing configuration.nix:
+<details>
+<summary>
+
+Getting started from scratch
+
+</summary>
```

Comment:
Please collapse this into a single line; it renders as a `<p>` on GitHub and adds a weird line break.

Reply:
Updated :+1: